assets/app/scripts/services/builds.js

```diff
@@ -59,7 +60,7 @@ angular.module("openshiftConsole")
     };
 
     // Function which will 'clone' build from given buildName
-    BuildsService.prototype.cloneBuild = function(buildName, $scope) {
+    BuildsService.prototype.cloneBuild = function(buildName, buildConfigName, $scope) {
```

Comment:
we dont always have a buildConfigName, you can create builds that arent part of a build config